packages/bruno-app/src/utils/importers/postman-collection.js

```diff
@@ -14,6 +14,56 @@ const readFile = (files) => {
   });
 };
 
+/**
+ *
+ * @param {string} query
+ * @returns {string}
+ */
+const parseGraphQLQuery = (query) => {
+  return query
+    .replace(/\s+/g, ' ')
+    .replace(/{ /g, '{')
+    .replace(/ {/g, '{')
+    .replace(/ }/g, '}')
+    .replace(/, /g, ',')
+    .replace(/ : /g, ': ')
+    .replace(/\n/g, '');
+};
+
+const parseGraphQLVariables = (string) => {
+  const cleanedString = string.replace(/[\n\t]/g, '').replace(/\\"/g, '"');
```

Comment:
Same here, don't think we should be stripping whitespace.

Reply:
Removed